src/Components/Slide/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import Decisions from '../Decisions';
+import Status from '../Status/index';
+import Icon from '../Icon/index';
+import './style.css';
+
+const business = {
```

Comment:
@tackc will this get passed in as a prop in the future? 
if so, maybe we add a `TODO: remove this obj once API wired up` or somethin?

also `function Slide(business) { ...`